src/hotspot/share/code/codeCache.cpp

```diff
@@ -356,6 +356,7 @@ void CodeCache::initialize_heaps() {
   }
 }
 
+
```

Comment:
You can undo that to avoid changing file.

Reply:
Done